Dockerfile

```diff
@@ -29,7 +29,7 @@ RUN apt-get update && \
 
 # Install the package manager - pipenv
 RUN pip install --upgrade pip && \
-    pip install --no-cache-dir pipenv
+    pip install --no-cache-dir pipenv pipenv==2021.5.29
```

Comment:
I think you want to drop the first copy of the word `pipenv` on all the lines you've changed.